android/settings.gradle

```diff
@@ -1,4 +1,6 @@
 rootProject.name = 'LNDR'
+include ':react-native-splash-screen'
+project(':react-native-splash-screen').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-splash-screen/android')
```

Comment:
Is the splash screen package from my work or is it a part of this privacy policy?